docs/src/pages/demos/text-fields/text-fields.md

```diff
@@ -57,9 +57,10 @@ but you are not confident jumping in, here's an example of how you can change th
 
 {{"demo": "pages/demos/text-fields/CustomizedInputs.js"}}
 
-Customization does not stop at CSS, you can use composition to build custom components and give your app a unique feel.
+Customization does not stop at CSS, you can use composition to build custom components and give your app a unique feel. It's demonstrating a use case for the [`InputBase`](/api/input-base/) component inspired by Google Maps.
```

Comment:
@oliviertassinari Thanks for the tip, the only thing I would say is the second part here should probably be re-worded. Perhaps 'Below is an example using the InputBase component, inspired by Google Maps'

Reply:
Sure, sounds good to me 👍